src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -418,6 +421,40 @@ public Vector4 CornerRadius
             set => SetValue(CornerRadiusPolicyProperty, value);
         }
 
+        /// <summary>
+        /// The width for the borderline of the View.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float? BorderlineWidth
+        {
+            get => (float?)GetValue(BorderlineWidthProperty);
```

Comment:
Why do you use nullable object(`float?`)? 
GetValue method never return null object, because it is defined as `float` not `float?`

``` c#
        public static readonly BindableProperty BorderlineWidthProperty = BindableProperty.Create(nameof(BorderlineWidth), typeof(float), typeof(ViewStyle), default(float), propertyChanged: (bindable, oldValue, newValue) =>
        {
            ((ViewStyle)bindable).borderlineWidth = (float)newValue;
        }, defaultValueCreator: (bindable) => ((ViewStyle)bindable).borderlineWidth);
```

Reply:
Oh, I miss that points. I'll make GetValue method retulrn `float?` type.